src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -263,6 +264,14 @@ public void configure(Context context) {
                     + "properly work!");
         } // if else
         
+        String csvSeparator = context.getString("csv_separator", ",");        
+        if (csvSeparator != null && csvSeparator.length() > 0) {
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/d89500c46c3c725a627c2b47e0fe6214cef0d6ad